pom.xml

```diff
@@ -11,7 +11,7 @@
     </parent>
     <groupId>CMAT</groupId>
     <artifactId>MeasureAuthoringTool</artifactId>
-    <version>6.11</version>
+    <version>6.12</version>
```

Comment:
Unless I'm overruled, I'll push for retaining the 6.11 version since the output of the tool is unchanged.